src/tools/merssh/command.cpp

```diff
@@ -253,6 +253,8 @@ void Command::maybeDoCMakePathMapping()
         QString data = QString::fromUtf8(reader.data());
         data.replace(Sfdk::Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT, sharedHomePath());
         data.replace(Sfdk::Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT, sharedSourcePath());
+        data.replace("/usr/bin/gcc", sdkToolsPath() + "/gcc");
+        data.replace("/usr/bin/cmake", sdkToolsPath() + "/cmake");
```

Comment:
Beautiful!

I think it's worth being more strict here, using "=/usr/bin/cmake$" instead and adding some QTC_CHECK here that it matched.

I checked CMakeCache.txt in one of test projects I built before and it containe /usr/bin/c++ but no /usr/bin/gcc.

Reply:
Doing what you suggest would require using regexp replace instead of simple string compare. But while making that change I could actually change it so that it replaces all CMAKE_CXX_COMPILER and CMAKE_C_COMPILER values, regardless of the original value. That should work even if the original cache file has /usr/bin/c++ or /usr/bin/cc.

Reply:
good idea, same could be done with cmake.